### PR TITLE
Updating sharing options

### PR DESCRIPTION
**Pull request by EdgarVerona**
So that people don't stumble on this tutorial first and get its instructions confused with the rules for posting on the Blueprint Exchange (as I did!)

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by EdgarVerona** [comment]
Ah, glad you had those automatic pull request comments!  I reviewed my change and removed my double spaces!

File: source/_docs/blueprint/tutorial.markdown [hunk 1]
```diff
@@ -227,3 +231,7 @@ The final step is to share this blueprint with others. For this tutorial we're g
 - Click the "Create Gist" button
 
 You can now copy the URL of your new Gist and share it with other people. They can import it by going to **Configuration**, **Blueprints** and clicking on the blue "Import Blueprint" button.
+
+### Share on the Blueprint Exchange
+
+If you follow the [Rules and format for posting](https://community.home-assistant.io/t/about-blueprints/253788), you can share your blueprint on the Home Assistant Blueprint Exchange forum. This option is accessible to the general Home Assistant community, but recommended only for your original blueprints. Please don't post this tutorial to the Blueprint Exchange, but instead remember this as an option for releasing your real blueprints.
```
Comment (frenck) [inline source/_docs/blueprint/tutorial.markdown]:
```suggestion
If you follow the [Rules and format for posting](/get-blueprints), you can share your blueprint on the Home Assistant Blueprint Exchange forum. This option is accessible to the general Home Assistant community but recommended only for your original blueprints. Please don't post this tutorial to the Blueprint Exchange, but instead, remember this as an option for releasing your real blueprints.
```

File: source/_docs/blueprint/tutorial.markdown [hunk 2]
```diff
@@ -218,7 +218,11 @@ Don't forget to reload automations after you make changes to your blueprint to h
 
 ## Share the love
 
-The final step is to share this blueprint with others. For this tutorial we're going to share it on GitHub Gists.
+The final step is to share this blueprint with others. 
+
+### Informal Sharing
+
+For this tutorial we're going to share it on GitHub Gists. This is a good option if you don't want to publish your blueprint to a larger audience.
```
Comment (frenck) [inline source/_docs/blueprint/tutorial.markdown]:
```suggestion
For this tutorial, we're going to share it on GitHub Gists. This is a good option if you don't want to publish your blueprint to a larger audience.
```

File: source/_docs/blueprint/tutorial.markdown [hunk 3]
```diff
@@ -218,7 +218,11 @@ Don't forget to reload automations after you make changes to your blueprint to h
 
 ## Share the love
 
-The final step is to share this blueprint with others. For this tutorial we're going to share it on GitHub Gists.
+The final step is to share this blueprint with others.
```
Comment (frenck) [inline source/_docs/blueprint/tutorial.markdown]:
```suggestion
The final step is to share this blueprint with others.
```

**Review by frenck — APPROVED** [review]
Thanks! 👍

**Comment by EdgarVerona** [comment]
Sweet, thank you! <3

**Pull request merged**